specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/vulnerabilityAssessment.json

```diff
@@ -0,0 +1,397 @@
+                    "description": "The identifier of a finding",
+                    "type": "string"
+                },
+				"title": {
+                    "description": "Finding's title",
+                    "type": "string"
+                },
+				"finding_type": {
+                    "description": "Finding's type",
+                    "type": "string"
+                },
+				"category": {
+                    "description": "Finding's category",
+                    "type": "string"
+                },
+				"severity": {
+                    "description": "Finding's severity",
+                    "type": "string"
+                },
+				"published_timestamp": {
+                    "description": "Time where the vulnerability was first published",
+                    "type": "string"
+                },
+				"cvss": {
+                    "description": "List of CVSS relevant for this finding",
+                    "type": "array",
+					"items": {
+                        "type": "string"
+					}
+                },
+				"patchable": {
+                    "description": "Is patchable",
+                    "type": "string"
+                },
+				"cve": {
+					"description": "The list of CVE relevant for this finding",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/LinkItem"
+					}
+				},
+				"threat": {
+                    "description": "The threat that this finding poses to the resource",
+                    "type": "string"
+                },
+				"solution": {
+                    "description": "Recommended Solution to Fix this Vulnerability",
+                    "$ref": "#/definitions/LinkItem"
+                },
+                "vendor_reference": {
+					"description": "Reference for the vendor",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/LinkItem"
+					}
+				},
+				"detection_type": {
+                    "description": "The type of detection",
+                    "type": "string"
+                },
+				"result": {
+                    "description": "A descriptive text of the result regarding this specific finding",
+                    "type": "string"
+                }
+			}
+        },
+        "VulnerabilityAssessment": {
+            "description": "Describes the status of a vulnerability assessment on a resource",
+            "properties": {
+				"properties": {
+					"description": "Properties of the Vulnerability Assessment resource",
+					"x-ms-client-flatten": true,
+					"properties": {
+						"status": {
+							"description": "The actual status",
+							"type": "string"
+						}
+					}
+				}
+            },
+			"allOf": [
+                {
+                    "$ref": "../../../common/v1/types.json#/definitions/Resource"
+                }
+            ]
+        },
+        "VulnerabilityAssessmentFindings": {

```

Comment:
resources must follow ARM conventions
you can look at any other collection resources that we have